apps/src/templates/teacherDashboard/SectionTable.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { Component, PropTypes } from 'react';
+import color from "@cdo/apps/util/color";
+import SectionRow, { sectionShape } from './SectionRow';
+
+const styles = {
+  table: {
+    width: '100%',
+    tableLayout: 'auto',
+    backgroundColor: 'transparent',
+    borderSpacing: 0,
+    borderCollapse: 'collapse'
+  },
+  headerRow: {
+    height: 45,
+    paddingTop: 5,
+    paddingBottom: 5,
+    paddingLeft: 15,
+    paddingRight: 15,
+    backgroundColor: color.teal,
+    color: color.white,
+    borderColor: color.white,
+    borderWidth: 1,
+    borderStyle: 'solid',
+    borderTopColor: color.teal,
+    borderLeftColor: color.teal,
+    borderRightColor: color.teal,
+    textAlign: 'left'
+  },
+  link: {
+    color: color.white
+  }
+};
+
+export default class SectionTable extends Component {
+  static propTypes = {
+    sections: PropTypes.arrayOf(sectionShape).isRequired
+  };
+
+  render() {
+    const { sections } = this.props;
+    // TODO: i18n
+    return (
+      <table style={styles.table}>
+        <colgroup>
+          <col width="200"/>
+          <col width="130"/>
+          <col width="98"/>
+          <col width="130"/>
+          <col/>
+          <col/>
+          <col/>
+        </colgroup>
```

Comment:
Not totally clear what we're doing with `colgroup` here, but I think we either need to add two `<col/>` to bring the total up to 9, or remove the tags that don't set a width because they're no-ops right now.